src/couch_views/src/couch_views_util.erl

```diff
@@ -276,3 +277,27 @@ is_paginated(#mrargs{page_size = PageSize}) when is_integer(PageSize) ->
 
 is_paginated(_) ->
     false.
+
+
+active_tasks_info(ChangesDone, DbName, DDocId, LastSeq, DBSeq) ->
+    #{
+        <<"type">> => <<"indexer">>,
```

Comment:
I still don't understand why we need this full active task map in jobs? The only values that we don't already store in the jobs data is `DbSeq` and `ChangesDone`. Couldn't you move those two fields into the job_data and then when you call `fabric2_active_tasks:get_active_tasks` create the full map to return to the active task info based off the job data. This map is a duplication of everything in the job data.

Reply:
I think the idea here is that if have different types of tasks, we would need to know what kind of fields in the JobData to extract for each new task. DBSeq and ChangesDone is probably universal, but different tasks might have unique fields and would then require an update to fabric_active_tasks each time.

Ex: Replication has  
   "checkpointed_source_seq": 68585,
   "continuous": false

and probably has other fields we want as part of active tasks, but we'd have to extract it in fabric2_active_tasks also. 

If we keep one standard active_tasks_info across all JobData, then it enforces encapsulation. As Nick showed up, it's only ~1KB.

Reply:
Ok that makes sense. We can maybe revisit that once we have active tasks for replication and see if its work slimming down.